types/src/encoding.rs

```diff
@@ -27,6 +28,8 @@ pub struct DecodingLimits {
     pub max_byte_string_length: usize,
     /// Maximum number of array elements. 0 actually means 0, i.e. no array permitted
     pub max_array_length: usize,
+    /// Time offset between the client and the server
```

Comment:
Can you rename offset something a little more clear and document it only needs to be used from the client side

Reply:
We can call it `client_offset` here as well? And then add a line to the comment as well. Would that work?

Reply:
Went ahead and added a commit that renames the field to `client_offset` and extends the comment.